src/integrationTest/java/uk/gov/hmcts/reform/userprofileapi/integration/RetrieveUserProfileInternalServerErrorIntTest.java

```diff
@@ -60,15 +60,15 @@ public void setUp() {
     public void should_return_500_and_not_create_user_profile_when_idam_service_throws_exception() throws Exception {
 
         when(idamService.registerUser(any(IdamRegisterUserRequest.class)))
-            .thenThrow(new RuntimeException("This is a test exception"));
+                .thenThrow(new RuntimeException("This is a test exception"));
 
```

Comment:
why the message is "This is a test exception"

Reply:
I have updated the message to reflect the exception being thrown, in this case Runtime Exception, in this commit: https://github.com/hmcts/rd-user-profile-api/pull/249/commits/144afe4b496160730449f10772744c491b34d1cb